subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheBuildTreeStructureIntegrationTest.groovy

```diff
@@ -285,19 +285,19 @@ class ConfigurationCacheBuildTreeStructureIntegrationTest extends AbstractConfig
             size() == 3
             it.find { it.details.buildPath == ":" }
             it.find { it.details.buildPath == ":include" }
-            it.find { it.details.buildPath == ":inner-include" }
+            it.find { it.details.buildPath == ":include:inner-include" }
```

Comment:
This is a breaking change, as the paths used to reference tasks in the inner build on the command-line will be different. We might be able to live with this breakage, however.

Reply:
Yeah, I think we can get away with the breakage. I added something to the upgrading notes.

Reply:
Should we also add something to the release notes?

Reply:
Given the size of the change I think it is enough that https://github.com/gradle/gradle/issues/17228 shows up under "Fixed issues" in the release notes.